force-app/main/default/objects/Tv_Serie__c/compactLayouts/Prototype2_Compact_Layout.compactLayout-meta.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
do you use this compact layout? It's name suggest that it was just a test version. If you use it - don't call it 'prototype'. If you don't use it - remove it from repo